crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -1,3 +1,26 @@
+//! Entity handling types.
+//!
+//! In Bevy ECS, there is no monolithic data structure for an entity. Instead, the [`Entity`]
+//! `struct` is just an identification number that [`Component`s](crate::component::Component)
```

Comment:
This could use some rephrasing:

1. Entity is not "just an identification number". It is a combination of an ID and a generation. This is often called a "generational index".
2. Components don't refer to Enitity ids. Entity ids map to specific components.